Okta.Core.Tests/Clients/UsersClientTests.cs

```diff
@@ -464,24 +484,40 @@ public void GetUserByUsername()
             catch (OktaException e)
             {
                 string strEx = string.Format("Error Code: {0} - Summary: {1} - Message: {2}", e.ErrorCode, e.ErrorSummary, e.InnerException.InnerException.Message);
-                //Console.WriteLine(strEx);
+                Console.WriteLine(strEx);
             }
-
         }
 
- 
-        //private Tenant getTenant(TestContext context)
-        //{
-        //    Tenant tenant = new Tenant
-        //    {
-        //        Id = Convert.ToInt32(context.DataRow["Id"]),
-        //        Url = Convert.ToString(context.DataRow["Url"]),
-        //        ApiKey = Convert.ToString(context.DataRow["ApiKey"])
+        [TestMethod]		
+         public void SetCredentials()
+        {
+            string strUserID = oktaTenant.TestUserId;
 
-        //    };
+            try
+            {
+                var usersClient = oktaClient.GetUsersClient();
 
-        //    return tenant;
-        //}
+                if (!string.IsNullOrWhiteSpace(strUserID))
+                {
+                    Models.User existingUser = usersClient.Get(oktaTenant.TestUserId);
+
+                    Assert.IsFalse(existingUser == null, "The user with id {0} doesn't exist", strUserID);
+
+                    Models.LoginCredentials loginCreds = new Models.LoginCredentials();
+                    loginCreds.Password.Value = Helpers.GetRandomString();
+
+                    Models.User updatedUser = usersClient.SetCredentials(strUserID, loginCreds);
+
+                    Assert.IsNotNull(updatedUser, "The updated user is null so there likely was an error while updating his password.");
+
+                }
+            }
+            catch (OktaException e)
+            {
+                string strEx = string.Format("Error Code: {0} - Summary: {1} - Message: {2}", e.ErrorCode, e.ErrorSummary, e.InnerException.InnerException.Message);
+                //Console.WriteLine(strEx);		

```

Comment:
Similar comment to @marksztainbok-okta's earlier - should an exception cause a failure? And, remove commented lines.